src/Staking.sol

```diff
@@ -96,21 +121,35 @@ contract StakeRegistry is AccessControl, Pausable {
     function manageStake(bytes32 _setNonce, uint256 _addAmount) external whenNotPaused {
         bytes32 _previousOverlay = stakes[msg.sender].overlay;
         bytes32 _newOverlay = keccak256(abi.encodePacked(msg.sender, reverse(NetworkId), _setNonce));
+        uint256 _addPotentialStake = _addAmount;
```

Comment:
Because I am obsessed with verbosely naming stuff, I added it so it's clear that we are adding the potential stake.

I changed it, let's see if I can survive it :)